demos/code/Blocks/Chapter 5 - Functions.js

```diff
@@ -1,179 +1,50 @@
 
 var funcHUE = 90;
 
-
-Blockly.Blocks['func_parameters'] = {
+Blockly.Blocks['function_declaration'] = {
```

Comment:
This is just a rename of the block. The reference in index.html has been updated as well.